application/wypl-core/src/main/java/com/wypl/wyplcore/calendar/service/CalendarService.java

```diff
@@ -0,0 +1,86 @@
+package com.wypl.wyplcore.calendar.service;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.wypl.googleoauthclient.domain.AuthMember;
+import com.wypl.jpacalendardomain.calendar.domain.Calendar;
+import com.wypl.jpacalendardomain.calendar.domain.MemberCalendar;
+import com.wypl.jpacalendardomain.calendar.domain.Schedule;
+import com.wypl.jpacalendardomain.calendar.repository.ScheduleRepository;
+import com.wypl.jpamemberdomain.member.domain.Member;
+import com.wypl.wyplcore.calendar.data.DateSearchCondition;
+import com.wypl.wyplcore.calendar.data.request.CalendarFindRequest;
+import com.wypl.wyplcore.calendar.data.response.CalendarSchedulesResponse;
+import com.wypl.wyplcore.calendar.service.strategy.CalendarStrategy;
+import com.wypl.wyplcore.schedule.data.CalendarType;
+import com.wypl.wyplcore.schedule.data.response.ScheduleFindResponse;
+import com.wypl.wyplcore.schedule.service.repetition.RepetitionService;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class CalendarService {
+
+	private final ScheduleRepository scheduleRepository;
+
+	private final Map<CalendarType, CalendarStrategy> calendarStrategyMap;
+
+	/**
+	 * 캘린더 정보와 캘린더의 일정을 함께 조회한다.
+	 * @param authMember : 인증된 사용자 정보
+	 * @param calendarId : 조회할 캘린더 ID
+	 * @param calendarFindRequest : 캘린더 조회 조건
+	 * @return FindCalendarResponse
+	 */
+	@Transactional
+	public CalendarSchedulesResponse findCalendar(AuthMember authMember, long calendarId,
+		CalendarFindRequest calendarFindRequest) {
```

Comment:
```suggestion
	public CalendarSchedulesResponse findCalendar(
		AuthMember authMember,
		long calendarId,
		CalendarFindRequest calendarFindRequest
	) {
```

이런 형식으로 매개변수를 작성해보는건 어떨까요?!